cluster/terraform_kubernetes/config/production.tfvars.json

```diff
@@ -103,6 +105,15 @@
     },
     "git-production/getintoteachingapi-production": {
        "receiver":"SLACK_WEBHOOK_GIT"
+    },
+    "cpd-production/cpd-tsh-production": {
+      "receiver":"SLACK_WEBHOOK_TSH"
+    },
+    "cpd-production/npq-registration-production-web": {
+      "receiver":"SLACK_WEBHOOK_NPQ"
+    },
+    "cpd-production/npq-registration-production-worker": {
```

Comment:
Are you sure we want to monitor the worker? Is this what devs want?
Workers may use 100% CPU as standard.